themes/custom/apigee_kickstart/templates/user/user--full.html.twig

```diff
@@ -8,7 +8,10 @@
 {% block content %}
   {# @todo Find a good way to constrict this to col-lg-6 that works with all
```

Comment:
Your CSS fix above completes this todo, so let's remove it. ;)

Reply:
Done. Thanks.